st2common/bin/migrations/v3.5/BUILD

```diff
@@ -1 +1,3 @@
+# TODO: what to do about st2-migrate-db-dict-field-values ?
```

Comment:
Is this to be resolved in another PR? Just checking this TODO is still relevant.

Reply:
Yeah. The TODO is still relevant. I'm not sure how best to handle the symlink - so I'm punting. :shrug: